src/commands/branch.rs

```diff
@@ -268,10 +268,17 @@ fn cmd_branch_delete(
     let names: BTreeSet<String> = args.names.iter().cloned().chain(globbed_names).collect();
     let branch_term = make_branch_term(names.iter().collect_vec().as_slice());
     let mut tx = workspace_command.start_transaction(&format!("delete {branch_term}"));
+    let mut count = 0;
     for branch_name in names {
+        if tx.mut_repo().get_local_branch(&branch_name).is_some() {
+            count += 1;
+        }
         tx.mut_repo().remove_local_branch(&branch_name);
     }
     tx.finish(ui)?;
+    if count > 1 {
+        writeln!(ui, "Deleted {count} branches.")?;
+    }
```

Comment:
Is there a case that `count != names.len()`?

Reply:
No, there really shouldn't be, thanks!